mtags/src/main/scala/scala/meta/internal/mtags/CommonMtagsEnrichments.scala

```diff
@@ -506,7 +506,7 @@ trait CommonMtagsEnrichments {
 
     def isScalaOrJava: Boolean = {
       toLanguage match {
-        case Language.SCALA | Language.JAVA => true
+        case Language.SCALA | Language.JAVA => path.toNIO.toFile().isFile()
```

Comment:
if we're with NIO then maybe `!Files.isDirectory(path.toNIO)` or `Files.isRegularFile(path.toNIO)`

Reply:
Good idea! I also added a check if the file exists because in that case we can only guess if it will be a file.